src/detectors/svelte.js

```diff
@@ -0,0 +1,36 @@
+const {
+  hasRequiredDeps,
+  hasRequiredFiles,
+  getYarnOrNPMCommand,
+  scanScripts
+} = require("./utils/jsdetect");
+
+module.exports = function() {
+  // REQUIRED FILES
+  if (!hasRequiredFiles(["package.json"])) return false;
+  // REQUIRED DEPS
+  if (!hasRequiredDeps(["svelte"])) return false;
+
+  /** everything below now assumes that we are within vue */
+
+  const possibleArgsArrs = scanScripts({
+    preferredScriptsArr: ["dev", "start", "run"],
+    preferredCommand: "npm run dev"
+  });
+
+  if (possibleArgsArrs.length === 0) {
+    // ofer to run it when the user doesnt have any scripts setup! 🤯
+    possibleArgsArrs.push(["npm", "dev"]);
+  }
+
+  return {
+    type: "svelte",
+    command: getYarnOrNPMCommand(),
+    port: 8888,
+    proxyPort: 5000,
+    env: { ...process.env },
+    possibleArgsArrs,
+    urlRegexp: new RegExp(`(http://)([^:]+:)${5000}(/)?`, "g"),
+    dist: "dist"
```

Comment:
I know this PR is merged already, but I was just wondering about the choice of `dist` here. The default Svelte template uses a `public` folder for its static output.

Reply:
i'm not that familiar with svelte, i didnt really check it. i have never liked the choice of `dist` as the keyword, as we dont care about *output*, we just care about the location of the `_redirects` file if it exists. this is a problem we'll have to fix at source

Reply:
Thanks for looking at this. The sapper location would be static which you updated correctly, but the Svelte default is `public`.